rectangle.py

```diff
@@ -1,8 +1,37 @@
+import random 
+
 tuple_list = []
-for w in range(2,10):
-  for h in range(3,10):
-    tuple_list.append((w,h))
 
-sorted_list = sorted(tuple_list, key=lambda tup: (tup[0] - 1) * (tup[1] - 1))
+start = 9
+
+for i in range(start,1,-1):
+  for j in range(start,1,-1):
```

Comment:
CLOSE!  Instead of 1 on this row, we want it to be shifting because when we look below, we only have 1 row of tup[0] 9, 2 rows of tup[0] 8, so on and so forth.  So we want the end (in our case) to be shifting.